blog.html

```diff
@@ -2,24 +2,84 @@
+						<li class="links"><a class="navlinks" href="https://github.com/ga-students/FEWD-DC-22/"> FAQ</a></li>
+						<li class="links"><a class="navlinks" href="https://github.com/ga-students/FEWD-DC-22/"> Team</a></li>
+						<li class="links"><a class="navlinks" href="https://github.com/ga-students/FEWD-DC-22/"> Contact Us</a></li>
+						<li class="links"><a class="navlinks" href="file:///C:/Users/wildcat/Documents/GitHub/Assignment%204/FEWD22-Assignment4/blog.html">Blog</a></li>
+					</ul>
+				</nav>
 			</header>
-			<footer>
-				&copy; Footer Goes Here
+			<div class="clearfix holder">
+				<main class="content">
+					<h2 class="title">How I implemented Relaxr in 2 weeks and changed my life</h2>
+					<img src="images/blog_photo1.jpg" alt="Blog Photo 1" class="photos">
+					<p class="post">Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea commodo consequat. Duis autem vel eum iriure dolor in hendrerit in vulputate velit esse molestie consequat, vel illum dolore eu feugiat nulla facilisis at vero eros et accumsan et iusto odio dignissim qui blandit praesent luptatum zzril delenit augue duis dolore te feugait nulla facilisi. </p>
+					<p><a class="more" href="https://github.com/ga-students/FEWD-DC-22/"> Read more ></a></p>
+					<hr class="clear">
+
+					<h2 class="title">I traveled across the globe thanks to Relaxr's Excel automation</h2>
+					<img src="images/blog_photo2.jpg" alt="Blog Photo 2" class="photos">
+					<p class="post">Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea commodo consequat. Duis autem vel eum iriure dolor in hendrerit in vulputate velit esse molestie consequat, vel illum dolore eu feugiat nulla facilisis at vero eros et accumsan et iusto odio dignissim qui blandit praesent luptatum zzril delenit augue duis dolore te feugait nulla facilisi. </p>
+					<p><a class="more" href="https://github.com/ga-students/FEWD-DC-22/"> Read more ></a></p>
+					<hr class="clear">
+				</main>
+
+
+				<aside class="extras">
+					<h3 class="sidetitle">Categories</h3>
+					<ul class="otherlinks">
+						<li>Success stories</li>
+						<li>Stats</li>
+						<li>How tos</li>
+						<li>Best of</li>
+						<li>Business tips</li>
+					</ul>
+
+					<h3 class="sidetitle">About Relaxr</h3>
+					<p class="about">Proin sed justo vel sapien varius auctor. Proin scelerisque massa at luctus tincidunt. Fusce porttitor, sapien sed tincidunt fringilla.</p>
+					<p class="learn"><a class="learn" href="https://github.com/ga-students/FEWD-DC-22/"> Learn more</a></p>
+					<img src="images/ad-unit.png" alt="Ad Unit"
```

Comment:
Make sure you close this tag! Using an HTML validator or relying on the color-coding in your text editor would help you avoid this.